src/main/java/com/soundcu/ofac/model/Entry.java

```diff
@@ -14,6 +15,11 @@
 	private List<String> alt_names;
 	private String call_sign;
 	private List<String> citizenships;
+	//This field was ignored because the data is coming in two different formats.
+	//A list of Strings and a String. That causes an exception on the mapping from JSON to Entities.
+	//"dates_of_birth": "1948"
```

Comment:
@ramirogm 

- I think I may create a custom mapping and then decide which instance has that value. If that instance is a String I can ignore it otherwise I will add it.  I need to test it and review if it works as expected.

* Another option would be mapping that field with this annotation
 
```
@JsonFormat(with = JsonFormat.Feature.ACCEPT_SINGLE_VALUE_AS_ARRAY)
private List<String> dates_of_birth;
```

So if the mapping finds a `String` instead of a `List`, will create an array with one element. I tested it and worked ok.

Please see this link as a reference. https://stackoverflow.com/questions/39041496/how-to-enforce-accept-single-value-as-array-in-jacksons-deserialization-process


* Leave it as it is. This field is not used right now. So it could be skipped. I think if it would need it, then I found the correct solution. Also, I think the service could be fixed soon the data which are wrong.


I'm comfortable with the last option. Let me know which option (or maybe you have another suggestion) can be implemented.

Reply:
done. thanks